service/gradle/testing.gradle

```diff
@@ -105,3 +105,29 @@ task azureTest(type: Test) {
     failOnPassedAfterRetry = true
   }
 }
+
+task azureUnitTest(type: Test) {
+  useJUnitPlatform {
+    includeTags "azure-unit"
+  }
+  outputs.upToDateWhen { false }
+  finalizedBy tasks.combinedJaCoCoReport
+
+  // For GHA runs, retry failing tests. If 2nd time passes, workflow will still
+  // fail and oncall will still be notified. This way, oncall dosen't have to
+  // rerun workflow in GHA.
+  retry {
```

Comment:
its unit test. do we need retry?

Reply:
No, I don't think so. I was just cutting and pasting. Will remove.